source/reference/minio-mc-admin/mc-admin-config.rst

```diff
@@ -2106,7 +2106,7 @@ configuration settings.
 
       *Optional*
 
-      .. include:: /includes/common-minio-external-auth.rst
+]      .. include:: /includes/common-minio-external-auth.rst
```

Comment:
I feel like this open bracket is out of place.

Reply:
That is a Ctrl + ] whoopsie